src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/OpenSslPkcs12Reader.cs

```diff
@@ -22,10 +22,7 @@ private OpenSslPkcs12Reader(SafePkcs12Handle pkcs12Handle)
 
         public unsafe static bool TryRead(byte[] data, out OpenSslPkcs12Reader pkcs12Reader)
```

Comment:
Do these methods that no longer need to call OpenSslD2I still need to be declared unsafe?